docker-compose.yml

```diff
@@ -6,10 +6,10 @@ version: '3'
 
 services:
   artemis-server:
-    command: sh -c "(apt update && apt install -y fontconfig ttf-dejavu graphviz || true) && ./gradlew buildJarForDocker && java -jar --add-exports java.naming/com.sun.jndi.ldap=ALL-UNNAMED build/libs/Artemis-*.jar"
+    command: sh -c "(apt update && apt install -y fontconfig fonts-dejavu graphviz || true) && ./gradlew buildJarForDocker && java -jar --add-exports java.naming/com.sun.jndi.ldap=ALL-UNNAMED build/libs/Artemis-*.jar"
     depends_on:
       - artemis-mysql
-    image: openjdk:17-jdk-buster
+    image: eclipse-temurin:17-jdk-jammy
```

Comment:
I would not specify the distribution here

Reply:
Thought it would be easier to spot EOL and the linux distribution this way even if jammy is the default eclipse temurin docker base image. For alpine see the comment below.

Reply:
I’d favour removing the distro name as well. To be able to spot that it is outdated, someone needs to know the Ubuntu codenames, notice that it is an old LTS version, and then a manual search and replace is needed to consistently update the base image everywhere.

Without a distro name we could be sure that always a maintained base image is used without having to do manual updates except when we switch to a new Java release anyway.